.github/workflows/ml_tools.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build-linux:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Gross, what was going on here?

Reply:
Some workflow nonsense.  I got it figured out and reverted this back to `latest`